common/flatpak-context.c

```diff
@@ -2844,12 +2964,33 @@ flatpak_context_allows_features (FlatpakContext        *context,
   return (context->features & features) == features;
 }
 
+char *
+flatpak_context_devices_to_usb_list (GHashTable *devices,
+                                     gboolean    hidden)
+{
+  GString *list = g_string_new (NULL);
+  GHashTableIter iter;
+  gpointer value;
+
+  g_hash_table_iter_init (&iter, devices);
+  while (g_hash_table_iter_next (&iter, &value, NULL))
+    {
+      if (hidden)
+        g_string_append_printf (list, "!%s;", (const char *) value);
+      else
+        g_string_append_printf (list, "%s;", (const char *) value);
+    }
+
+  return g_string_free (list, FALSE);
```

Comment:
`g_string_free_and_steal`

Reply:
`g_string_free_and_steal` is from GLib 2.76, which is not available on Ubuntu 22.04 I think

Reply:
It has the same semantics as `g_string_free (list, FALSE);` but is more descriptive. Maybe we should just add it to glnx-backports?

Reply:
like 2 other places. Increasing the minimum glib isn't in scope. And nothing warrant me touching glnx-backports in this PR.